pkg/datapath/tables/device.go

```diff
@@ -100,6 +102,36 @@ func (d *Device) HasIP(ip net.IP) bool {
 	return false
 }
 
+func (*Device) TabHeader() []string {
```

Comment:
nit: We might want to change the naming here perhaps Table(Header/Row)?

Reply:
Changed.